code/drasil-example/nopcm/lib/Drasil/NoPCM/Body.hs

```diff
@@ -257,14 +258,19 @@ orgDocEnd = foldlSent_ [atStartNP (the inModel),
 --Section 3 : GENERAL SYSTEM DESCRIPTION
 ----------------------------------------
 
---ALL OF THIS SECTION IS NOW PULLED FROM SWHS
+--ALL OF THIS SECTION IS NOW PULLED FROM SWHS (Exept System Context Figure)
 
 --TODO: If/when system constraints recieves any content, add s3_3_intro
 
 ------------------------------
 --Section 3.1 : SYSTEM CONTEXT
 ------------------------------
 
+sysCntxtFig :: LabelledContent
+sysCntxtFig = llcc (makeFigRef "SysCon") $ fig (foldlSent_
+  [refS sysCntxtFig +: EmptyS, titleize sysCont])
```

Comment:
```suggestion
sysCntxtFig = llcc (makeFigRef "SysCon") $ fig (refS sysCntxtFig +: titleize sysCont)
```
This should work and is cleaner. Nice job finding the use of "SWHS" in the non-searchable image!

Reply:
This results in the colon being after `sysCont` I don't believe we have a constructor for the colon in between

Reply:
Ah yes, good catch